src/features/todos.ts

```diff
@@ -1,9 +1,29 @@
+/* eslint-disable import/no-cycle */
 import { Todo } from '../types/Todo';
 
-export const actions = {};
+type SetActionTodos = {
+  type: 'todos/set';
+  payload: Todo[];
+};
+
+const setTodos = (todos: Todo[]): SetActionTodos => ({
+  type: 'todos/set',
+  payload: todos,
+});
+
+type State = Todo[];
+type Action = SetActionTodos;
+
+const todosReducer = (state: State = [], action: Action): Todo[] => {
+  switch (action.type) {
```

Comment:
Consider adding a blank line before the switch statement for better readability.